src/en/troubleshooting-logs-audit.md

```diff
@@ -0,0 +1,28 @@
+Title: Audit logging
+
+# Audit logging
+
+Juju audit logging provides a chronological account of all events. These logs
```

Comment:
It captures all commands run by users (or scripts talking to the API) - it doesn't contain API calls by Juju agents. It might be clearer to say "all user commands" here?

Reply:
:ok: